v2/holiday.go

```diff
@@ -50,6 +50,7 @@ type Holiday struct {
 	Julian     bool         // the holiday is based on a Julian calendar
 	Observed   []AltDay     // the substitution days for the holiday
 	Func       HolidayFn    // logic used to determine occurrences
+	Lunar      bool         // the holiday is based on a Lunar calendar
```

Comment:
I don't see this used anywhere. Is it needed?

Reply:
some kr holidays uses `Lunar` flag. such as https://github.com/aanoaa/cal/blob/kr-holidays/v2/kr/kr_holidays.go#L45